### PR TITLE
Start QR code scanner from app

### PR DESCRIPTION
**Pull request by di72nn**
In a feat of procrastination I wrote some code for the app to invoke QR-code scanner.
I added a button on one of the wizard pages. Suggestions are welcome.

### CODE REVIEW

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java [hunk 1]
```diff
@@ -132,6 +169,25 @@ private ConnectionData parseLoginData(String connectionUri) {
         return new ConnectionData(values[0], values[1]);
     }
 
+    public void scanQrCode() {
+        try {
+            Intent intent = new Intent("com.google.zxing.client.android.SCAN");
```
Comment (tcitworld) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java]: Does that enforce a specific QR scanner client ?

**Comment by tcitworld** [comment]
Needs rebase to have only the second commit.

**Comment by di72nn** [comment]
It was a PITA to set up the app multiple times during development, so I updated this PR. I also added support for password. With these changes one can set up his account in one swift press-the-button-find-the-qr-code-point-the-phone-at-the-code motion.

The intent works with **multiple QR code scanning applications**: a list of apps is presented by Android. I can see "Barcode Scanner" and "Binary Eye" (I don't think I have any other installed).

**Comment by di72nn** [comment]
Added `intent.removeExtra(EXTRA_FILL_OUT_FROM_SETTINGS);` which I missed yesterday. Without it the feature didn't work if the wizard wasn't started from settings.

**Pull request merged**